vcl_templates/www.vcl.erb

```diff
@@ -296,6 +296,9 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-Example=" req.http.GOVUK-ABTest-Example "; expires=" now + 1d;
   }
 
+  # Set the tls version cookie with the raw protocol version from fastly
+  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; max-age=0; secure";
```

Comment:
re: `max-age=0` I think (and @davidillsley might correct me - I took it from his proof-of-concept) that our intention is to use cookies as a way of getting the value from the CDN into something that can be read by JS in the browser, not necessarily that we want to persist the cookie.  But perhaps we should be using `expires=" now` instead (better cross-browser support).

re: repercussions - I can't think of any other than the increase in payload size which will add up over time, but I have no idea what else we might want to measure or monitor.